plugin/rewrite/setup_test.go

```diff
@@ -22,4 +22,45 @@ func TestParse(t *testing.T) {
 	if err != nil {
 		t.Errorf("Expected success but found %s for `rewrite name a.com b.com`", err)
 	}
+
+	c = caddy.NewTestController("dns",
+		`rewrite stop {
+    name regex foo bar
+    answer name bar foo
+}`)
+	_, err = rewriteParse(c)
+	if err != nil {
+		t.Errorf("Expected success but found %s for valid response rewrite", err)
+	}
+
+	c = caddy.NewTestController("dns", `rewrite stop name regex foo bar answer name bar foo`)
+	_, err = rewriteParse(c)
+	if err != nil {
+		t.Errorf("Expected success but found %s for valid response rewrite", err)
+	}
+
+	c = caddy.NewTestController("dns",
+		`rewrite stop {
+    name regex foo bar
+    answer name bar foo
+    name baz qux
+}`)
+	_, err = rewriteParse(c)
+	if err == nil {
+		t.Errorf("Expected error but got success for invalid response rewrite")
+	} else if err.Error() != "response rewrites must consist only of a name rule with 3 arguments and an answer rule with 3 arguments" {
```

Comment:
it's better to use strings.Contain to check these errors strings so we have a less tight coupling between the tests and code